splunk_add_on_ucc_framework/global_config_update.py

```diff
@@ -51,6 +51,10 @@ def _handle_biased_terms(conf_entities: List[Dict[str, Any]]) -> List[Dict[str,
 def _handle_biased_terms_update(global_config: global_config_lib.GlobalConfig) -> None:
     for tab in global_config.tabs:
         conf_entities = tab.get("entity")
+
+        if conf_entities is None:
+            continue
```

Comment:
Can you please expand on this line a bit more?

Reply:
I was not sure if that change was necessary.

If `schemaVersion` is older, this code is executed and in case of a logging tab, `tab.get("entity")` returns `None` and due to that `_handle_biased_terms` throws an error. I thought about replacing `tab.get("entity")` with `tab.entity` but in case of a logging tab, this property is read-only.

Reply:
so this is to handle cases when user will define tab in a new way using:      
```
           {
                    "type": "loggingTab"
                }
```
Am I get it right?

Reply:
Yes. If the tab is defined in this format and "schemaVersion" is equal to 0.0.1. Maybe it should fail in such a case but I decided to propose such a change.